pkg/scripts/environment_test.go

```diff
@@ -3,6 +3,8 @@ package scripts
 import (
 	"testing"
 
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/builder/dockerfile"
```

Comment:
A bit confused why we need to pin our dependency on `moby/buildkit` when we don't reference it in the test.

Reply:
It's a dependency of the dockerfile module.